examples/configurations/api/groq_api.py

```diff
@@ -0,0 +1,19 @@
+import os
+from llama_index.llms.groq import Groq
+
+
+class GroqApiLLM(Groq):
+    # This class initializes the Groq API with selected models.
+    # Available models: llama3-8b-8192, llama3-70b-8192, llama2-70b-4096, mixtral-8x7b-32768, gemma-7b-it
+    def __init__(
+            self,
+            model: str = "llama3-70b-8192",
+            api_key: str = os.getenv("GROQ_API_KEY"),
+    ):
+        if api_key is None:
+            raise ValueError("GROQ_API_KEY environment variable is not set")
+
+        super().__init__(
+            model=model,
+            api_key=api_key,
+        )
```

Comment:
Can you add llama-index-llms-groq==0.1.3 to the pyproject.toml file too

Reply:
I've added the "llama-index-llms-groq==0.1.3" to the pyproject.toml file to ensure compatibility with LaVague.